src/pyansys_sphinx_theme/__init__.py

```diff
@@ -39,3 +41,122 @@ def setup(app):
         "parallel_read_safe": True,
         "parallel_write_safe": True,
     }
+
+
+latex_elements = {
```

Comment:
Let us create a new `src/pyansys_sphinx_theme/latex.py` module. This will be in charge of collecting all $\LaTeX$ constants, variables, and routines.

Think of a developer who would like to customize this within the limits of Ansys and PyAnsys guidelines:

```python
# File: conf.py
latex_elements = {
    "preamble": pyansys_sphinx_theme.latex.generate_preamble(project, watermark),
    ...
}
```

Previous function may look like:

```python
# File: pyansys_sphinx_theme/latex.py
def generate_preamble(project_name, watermark=DEFAULT_WATERMARK, ...):
    """Return an f-string containing the LaTeX code for the preamble."""
    ...
    return premble
```